src/Gui/Qt4All.h

```diff
@@ -168,12 +162,10 @@
 #include <QtDesigner/QFormBuilder>
 
 // QtWebKit
-#if QT_VERSION >= 0x040400
 // Only needed in Web module
 //#include <QWebFrame>
 //#include <QWebView>
 //#include <QWebSettings>
-#endif
 

```

Comment:
I think you can remove this whole section

Reply:
I think you're right, I don't see any good reason to leave that in.